vyxal/structure.py

```diff
@@ -15,20 +15,13 @@ def __init__(self, *branches: Branch):
         # Don't do anything with the arguments
         self.branches = branches
 
-    def transpile(self) -> str:
-        """Return the transpiled version of the structure"""
-        return "{}"
-
     def __repr__(self):
         return f"{type(self).__name__}({repr(self.branches)})"
 
 
 class GenericStatement(Structure):
     """Generic statements are elements and so on"""
 
-    def __init__(self, *branches: Branch):
-        super().__init__(*branches)
```

Comment:
Did we need these?

Reply:
No, `super().__init__` does the same thing, no need to overload. The other structures, however, do need overloads because they do special things with `branches`.